src/runtime/parallel/coordinator.ts

```diff
@@ -213,18 +250,67 @@ export default class Coordinator {
     _.times(numberOfWorkers, (id) =>
       this.startWorker(id.toString(), numberOfWorkers)
     )
-    this.onFinish = done
+    this.onFinish = (status) => {
+      if (this.idleInterventions > 0) {
+        console.warn(
+          `WARNING: All workers went idle ${this.idleInterventions} time(s). Consider revising handler passed to setParallelCanAssign.`
+        )
```

Comment:
Nice, I like this idea. I think using `console.warn` is sufficient. No need to send this to the formatters

Reply:
Can we add a feature tests to show this idleIntervention logic works as expected? (I think the current first feature test would print this)

Reply:
Having trouble figuring out how to test if something printed to `std.out`. You are absolutely right though, the first test prints this warning, so we just need to add onto that scenario. Once we figure out what that addition needs to be.

Reply:
@charlierudolph after a bit of a hiatus I finally got back to this and found the `spawn` tag which made it simple.